operator/pkg/tasks/init/crd.go

```diff
@@ -156,3 +180,17 @@ func existCrdsTar(crdsDir string) bool {
 	}
 	return false
 }
+
+func getCrdsDir(data InitData) (string, error) {
+	crdTarball := data.CrdTarball()
+	key := ""
+	if crdTarball.HTTPSource != nil {
+		key = crdTarball.HTTPSource.URL
+	} else {
+		return "", errors.New("[getCrdsDir] No CRD download strategy has been specified")
```

Comment:
The error message might be misleading with the download policy. How about that:

```suggestion
		return "", errors.New("[getCrdsDir] No CRD download source has been specified")
```
But this shouldn't happen, right? as we have default HTTP source.

Reply:
Good point. Pushed change to address.